packages/fxa-settings/src/components/Security/index.test.tsx

```diff
@@ -68,6 +68,7 @@ describe('Security', () => {
         passwordCreated: 1234567890,
         hasPassword: true,
       } as unknown as Account;
+      const createDate = new Date(1234567890).getDate();
```

Comment:
```sh
> new Date(1234567890) # 1970-01-15T06:56:07.890Z
> new Date(1234567890).getDate() # 14
```

Since [`Date#getDate()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getDate) returns the day of the month, isn't this always going to return `14`?

Reply:
I believe it will depend on the timezone of the system that does the call:
```
> new Date(1234567890)
# Date Wed Jan 14 1970 22:56:07 GMT-0800 (Pacific Standard Time)

> new Date(1234567890).getDate()
# 14 
```

Reply:
True, I guess we'd have to use [`Date.UTC()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC) if we wanted a consistent date.